### ISSUE TITLE
Use MINISHIFT_VM_DRIVER for integration tests

### ISSUE THREAD

**Issue by gbraad** [minishift/minishift#440]
The VM driver should be controlled by MINISHIFT_VM_DRIVER, instead of the VM_DRIVER in `flags.go`.

**Comment by gbraad** [comment]
```
MINISHIFT_VM_DRIVER=virtualbox minishift start                                                                                                                                                                              Starting local OpenShift cluster using 'virtualbox' hypervisor...
```

currently `VM_DRIVER` is used in the `centos_ci.sh` script, which controls a parameter, which else takes the default of `virtualbox` in `flags.go`, which is used in the integration tests.

**Comment by gbraad** [comment]
The change has to be done in `centos_ci.sh`... we should NEVER be using `VM_DRIVER`, but the already define `MINISHIFT_VM_DRIVER`.

**Comment by budhrg** [comment]
> The change has to be done in centos_ci.sh... we should NEVER be using VM_DRIVER, but the already define MINISHIFT_VM_DRIVER.

@gbraad 

Along with changes in `flags.go` https://github.com/minishift/minishift/blob/master/test/integration/flags.go#L36.

For me, it looks like we don't need `vmDriver` in `flags.go` at all.

**Comment by gbraad** [comment]
@budhrg It has already been removed in my changeset ;-)

**Comment by budhrg** [comment]
@gbraad  
> @budhrg It has already been removed in my changeset ;-)

Then, mention in the PR that it is also fixing this issue ;-)

**Comment by gbraad** [comment]
Added

**Comment by hferentschik** [comment]
Merged via pull request #433
